src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Text.txt

```diff
@@ -4,9 +4,6 @@
 >> Text(123.4, "#.000")
 "123.400"
 
->> Text(Blank(), "#.000")
```

Comment:
Not remove these tests, just move and update tests of this file of format string for non numeric input to Text_Format file.